doc/reference/internals/iproto/replication.rst

```diff
@@ -135,6 +163,70 @@ is an optional key used in the SUBSCRIBE request followed by an array
 of ids of instances whose rows won't be relayed to the replica.
 The field is encoded only when the ID list is not empty.
 
+..  _box_protocol-fetch-snapshot:
+
+IPROTO_FETCH_SNAPSHOT
+~~~~~~~~~~~~~~~~~~~~~
+
+Code: 0x45.
+
+To join a replica set as an anonymous replica, an instance must send an initial
+IPROTO_FETCH_SNAPSHOT request to any node in the replica set:
+
+..  raw:: html
+    :file: images/repl_fetch_snapshot_request.svg
+
+To learn about anonymous replicas, see :ref:`replication.anon <configuration_reference_replication_anon>`.
+
+..  include:: replication.rst
+    :start-after: iproto_join_response_sequence_start
+    :end-before: iproto_join_response_sequence_end
```

Comment:
A response for FETCH_SNAPSHOT is shorter than the one for JOIN:
it contains only
IPROTO_JOIN_META / IPROTO_JOIN_SNAPSHOT stage (optional)
and the IPROTO_INSERT stage (without replace,update, etc)